jukeInTheBoxUti/www/src/main.js

```diff
@@ -0,0 +1,53 @@
+import Accueil from './Accueil.vue'
+import File from './File.vue'
+import Biblio from './Bibliotheque.vue'
+
+const start = {template: `
+<div class="index">
+        <img src="./src/assets/logo.png" width="156.2px" height="156.2px" />
+        <h1>Juke in the box</h1>
+        <router-link to="/accueil">
+            <div class="home_button">
+                <p>Accéder à l'application</p>
+            </div>
+        </router-link>
+      </div>
+`}
+const routes = [
+  { path: '/', component: start },
+  { path: '/accueil', component: Accueil },
+  { path: '/bibliotheque', component: Biblio },
+  { path: '/file', component: File}
+]
+
+const router = new VueRouter({
+  routes 
+})
+
+const app = new Vue({
+  router,
+  data : {
+    url : "https://webetu.iutnc.univ-lorraine.fr/www/rimet2u/jukeinthebox/",
+    token :localStorage.qrcode,
+    unshow: true,
+  },
+  methods: {
+    //permet de valider la saisie d'un token
+    entrerToken : function(){
```

Comment:
Vous devriez gérer l'authentification directement dans les routes en question, et rediriger vers une page pour que l'utilisateur entre son token s'il n'est pas authentifié.

https://scotch.io/tutorials/vue-authentication-and-route-handling-using-vue-router
https://router.vuejs.org/guide/advanced/data-fetching.html